decavision/dataset_preparation/generate_pseudolabels.py

```diff
@@ -0,0 +1,159 @@
+import os
+import pandas as pd
+import tensorflow as tf
```

Comment:
In the imports the convention that I like to use is to have in general 3 blocks separated by empty lines. The first block is importing the packages that are included in basic python (like os). The second block is for installed packages (like pandas). The third block is for when you import code from other scripts that you wrote. In all of the blocks I think it's also nice to keep the imports in alphabetical order.